src/frontend/src/main.tsx

```diff
@@ -2,6 +2,26 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 
 import App from './App';
+import { HostList } from './states/states';
+
+// define settings
+declare global {
+  interface Window {
+    INVENTREE_SETTINGS: {
+      server_list: HostList;
+      default_server?: string;
+      show_server_selector: boolean;
+    };
+  }
+}
+
+window.INVENTREE_SETTINGS = {
+  server_list: {},
+  show_server_selector: true,
```

Comment:
Do you think this should be enabled by default? My suggestion would be also to remove the demo server from the host list, if you need it you can configure it. But if you want to leave it, feel free, then Ill do that only for my deployments via configuration.

Reply:
I think for development, it's fine. But *most* installs will want to keep the simplistic single-server approach by default. I think we should probably disable it for production release.

Reply:
This should be resolved now. If we are in dev mode, the demo server and the current url are the default. (That's also why I added the proxy option). And if we are not in dev, only the current path is in the host list which is also the default and we don't show the server selector.